<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hexsendbox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Hex发送</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hexshowbox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Hex显示</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="serialportLabel">
       <property name="text">
        <string>串口号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="portBox"/>
     </item>
     <item>
      <widget class="QLabel" name="baudrateLabel">
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="baudBox"/>
     </item>
     <item>
      <spacer name="serialPortLayout">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="openPortBtn">
       <property name="text">
        <string>打开串口</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="txData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>551</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="RxDataLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>接受数据：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="rxData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>641</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
